deepchem/molnet/load_function/material_datasets.py

```diff
@@ -0,0 +1,359 @@
+        DEFAULT_TRANSFORMERS['NormalizationTransformer']
+    ],
+    splitter: Splitter = DEFAULT_SPLITTERS['RandomSplitter'],
+    reload: bool = True,
+    data_dir: Optional[str] = None,
+    save_dir: Optional[str] = None,
+    featurizer_kwargs: Dict[str, object] = {'data_source': 'matminer'},
+    splitter_kwargs: Dict[str, object] = {
+        'frac_train': 0.8,
+        'frac_valid': 0.1,
+        'frac_test': 0.1
+    },
+    transformer_kwargs: Dict[str, Dict[str, object]] = {
+        'NormalizationTransformer': {
+            'transform_X': True
+        }
+    },
+    **kwargs) -> Tuple[List, Tuple, List]:
+  """Load band gap dataset.
+
+  Contains 4604 experimentally measured band gaps for inorganic
+  crystal structure compositions.
+  
+  Parameters
+  ----------
+  featurizer : ElementPropertyFingerprint
+    A featurizer that inherits from deepchem.feat.Featurizer.
+  transformers : List{List of allowed transformers for this dataset}
+    A transformer that inherits from deepchem.trans.Transformer.
+  splitter : RandomSplitter
+    A splitter that inherits from deepchem.splits.splitters.Splitter.
+  reload : bool (default True)
+    Try to reload dataset from disk if already downloaded. Save to disk
+    after featurizing.
+  data_dir : str, optional
+    Path to datasets.
+  save_dir : str, optional
+    Path to featurized datasets.
+  featurizer_kwargs : dict
+    Specify parameters to featurizer, e.g. {"size": 1024}
+  splitter_kwargs : dict
+    Specify parameters to splitter, e.g. {"seed": 42}
+  transformer_kwargs : dict
+    Maps transformer names to constructor arguments, e.g.
+    {"BalancingTransformer": {"transform_x":True, "transform_y":False}}
+  **kwargs : additional optional arguments.
+
+  Returns
+  -------
+  tasks, datasets, transformers : tuple
+    tasks : list
+      Column names corresponding to machine learning target variables.
+    datasets : tuple
+      train, validation, test splits of data as
+      ``deepchem.data.datasets.Dataset`` instances.
+    transformers : list
+      ``deepchem.trans.transformers.Transformer`` instances applied
+      to dataset.
+
+  References
+  ----------
+  .. [1] Zhuo, Y. et al. "Predicting the Band Gaps of Inorganic Solids by Machine Learning." J. Phys. Chem. Lett. (2018) DOI: 10.1021/acs.jpclett.8b00124.
+
+  .. [2] Dunn, A. et al. "Benchmarking Materials Property Prediction Methods: The Matbench Test Set and Automatminer Reference Algorithm." https://arxiv.org/abs/2005.00707 (2020)
+
+  Examples
+  --------
+  >> import deepchem as dc
+  >> tasks, datasets, transformers = dc.molnet.load_bandgap(reload=False)
+  >> train_dataset, val_dataset, test_dataset = datasets
+  >> n_tasks = len(tasks)
+  >> n_features = train_dataset.get_data_shape()[0]
+  >> model = dc.models.MultitaskRegressor(n_tasks, n_features)
+
+  """
+
+  # Featurize
+  logger.info("About to featurize band gap dataset.")
+  my_tasks = ['gap expt']  # machine learning targets
```

Comment:
And, could you clarify this column name...? I think the `experimental_bandgap` is better.